etc/cvmfs/common.conf

```diff
@@ -2,6 +2,11 @@
 # cvmfs does not parse included files for variable names, so first need
 #   to set in the domain config any variable set here.
 
-if [ -n "$CVMFS_HTTP_PROXY" ] && [ "$CVMFS_HTTP_PROXY" != "DIRECT" ] && [ "$CVMFS_HTTP_PROXY" != "auto;DIRECT" ]; then
-  CVMFS_FALLBACK_PROXY="http://cvmfsbproxy.cern.ch:3126;http://cvmfsbproxy.fnal.gov:3126"
```

Comment:
It is important to still include the fallback proxies when accessing the known grid domains, because they could be being used from a large site that has its own squids that are failing.  We do not want those sites connecting directly to stratum 1s with lots of worker nodes.

Since you have repurposed common.conf and you may want to use it from other domains sometime, you might want to instead put this in a new config file fallback.conf or something like that.